plugins/aea-test-autonomy/aea_test_autonomy/docker/amm_net.py

```diff
@@ -49,25 +48,21 @@
 class AMMNetDockerImage(DockerImage):
     """Spawn a local Ethereum network with deployed Gnosis Safe and Uniswap contracts, using HardHat."""
 
-    third_party_contract_dir: Path
-
     def __init__(
         self,
         client: docker.DockerClient,
-        third_party_contract_dir: Path,
         addr: str = DEFAULT_HARDHAT_ADDR,
         port: int = DEFAULT_HARDHAT_PORT,
     ):
         """Initialize."""
         super().__init__(client)
         self.addr = addr
         self.port = port
-        self.third_party_contract_dir = third_party_contract_dir
 
     @property
     def tag(self) -> str:
```

Comment:
we should consider renaming this to `image`. We return the image, not the tag.

Reply:
would be a breaking change - so suggest opening an issue instead

Reply:
https://github.com/valory-xyz/open-autonomy/issues/1340